docs/sync.js

```diff
@@ -720,6 +710,13 @@ class User {
      */
     invalidatePermissionOffer(permissionOfferOrToken) { }
```

Comment:
Shouldn't this be `invalidatePermissionsOffer` (note plural permissions). It seems to me that we are normally talking about multiple permissions when referring to the `Permission` object (my head hurts already). But to me, this means that these methods should also be plural, so `invalidatePermissionsOffer` and `getPermissionsOffers`